.github/workflows/release.yml

```
@@ -82,26 +81,30 @@ jobs:
         run: echo "${{ secrets.YC_OAUTH_TOKEN }}" | docker login --username oauth --password-stdin cr.yandex
 
       - name: Build & push STABLE
-        if: ${{ needs.release.outputs.is_beta == 'false' }}
+        if: ${{ needs.prepare-release-info.outputs.is_beta == 'false' }}
         run: |
-          TAG=${{ needs.release.outputs.version }}
-          docker build -t $REGISTRY:$TAG .
+          TAG=${{ needs.prepare-release-info.outputs.version }}
+          docker build \
+            --build-arg SENTRY_AUTH_TOKEN=${{ secrets.SENTRY_AUTH_TOKEN }} \ 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)